### ISSUE TITLE
Can't build project in Xcode 12

### ISSUE THREAD

**Issue by bulkinav** [parse-community/Parse-SDK-iOS-OSX#1533]
Parse 1.18.0

The following issue appears when I try to build project in Xcode 12 beta 5:

Pods/Parse/Core/ParseClientConfiguration.m:

```
(instancetype)copyWithZone:(NSZone *)zone {
    return [ParseClientConfiguration configurationWithBlock:^(ParseClientConfiguration *configuration)

```
Incompatible block pointer types sending 'void (^)(ParseClientConfiguration *__strong)' to parameter of type 'void (^ _Nonnull)(id<ParseMutableClientConfiguration>  _Nonnull __strong)'

**Comment by TomWFox** [comment]
This is a duplicate of #1526, the issue was fixed by #1527 which has been merged into the master branch.

**Comment by drdaz** [comment]
There's a fix for this in master. It'll be included in the next release.

If you can, use the current latest commit in master in the meantime.
